index.html

```diff
@@ -1183,6 +1183,110 @@
 browser fingerprint.
 
+## Information access {#information}
+
+The many APIs available to websites expose lots of data that can be combined into
+information about people, web servers, and other things. We can divide that
+information into three categories:
+
+1. Information that's fine to expose, for example because a person or group with
+    sufficient authority intended to expose that information or to do something that
+    necessarily exposes the information, or because it's not about people at all. For
+    example:
+
+    * The geolocation and camera APIs ask whether a person wants to expose their data.
+    * The URL a person is visiting must be sent to a server in order to navigate to that
+       URL, and known private-information-retrieval methods are too expensive to avoid that
+       exposure.
+    * The [[[largest-contentful-paint]]] distribution for a website is about a website
+       rather than about the people browsing it, even if the data that informs that
+       measure can also reveal information about people.
+
+1. Information that we don't want to expose and have a plausible plan for removing access
+    to. For example, browsers are gradually removing the ability to join identities
+    between different [=partitions=].
+
+1. Information that we'd rather not expose, but that we don't have a plausible plan for
+    removing access to. For example:
+
+    * Some users are disappointed that the page they're visiting can discover which link
+        they clicked to leave that page. We can't block that information that information
+        because the page can use <a data-cite="RFC9110#status.3xx">HTTP redirects</a> to
+        learn it, and redirection is a core feature of the web.
+    * Some users are disappointed that a page with permission to run Javascript can record
+        their pattern of interaction with that page. However, the page does this by using
+        the same events it would use to make the page interactive, so we can't block this
+        information access either.
+
+The following subsections discuss how to review an API proposal that exposes data that
+provides a new way to infer each of the above categories of information. They explain how
+to <a data-cite="design-principles#leave-the-web-better">leave the web better than you
+found it</a>.
+
+### Handling acceptable information {#acceptable-information}
+
+<div class="practice">
+  <span class="practicelab">New APIs can add additional ways of getting acceptable
+  information that are guarded at least as strongly as the old ways.</span>
+</div>
+
+Acceptable information exposure is always qualified by the (possibly empty) set of
+user-controlled settings or permissions that guard access to it. For example, the URLs of
+resources, the timing of link clicks, and the referrer chain within a single origin are
+not guarded by anything; the scroll position is guarded by the setting to turn off
```

Comment:
I don't think referrer is that simple? A new feature would have to at least abide by Referrer Policy (and some user agents might also allow disabling it always?).

Reply:
In existing UAs, are same-origin referrers actually guarded by any user setting or permission? Referrer Policy, for example, is controlled by the site, so it doesn't in itself constitute a user setting. This paragraph is trying to give readers a sense of how we have to keep existing guards in new features, and I don't think speculating about future UA behavior changes helps with that.